cg/services/application/service.py

```diff
@@ -0,0 +1,21 @@
+from cg.models.orders.constants import OrderType
+from cg.services.application.models import ApplicationResponse
+from cg.store.models import Application
+from cg.store.store import Store
+
+
+def create_application_response(app_tags: list[str]) -> ApplicationResponse:
+    return ApplicationResponse(applications=app_tags)
+
+
+class ApplicationsService:
```

Comment:
In light of the discussion @islean and I had in this PR (see the other comments). the aim is to reduce the scope of this service to only WebAPI interactions. A narrow scope means less code to write and less things to consider/ test. 

```suggestion
class ApplicationsWebService:
```

Reply:
I totally agree